backend/src/graphql/resolver/util/creations.ts

```diff
@@ -21,7 +21,7 @@ export const validateContribution = (
   if (index < 0) {
     logger.error(
       'No information for available creations with the given creationDate=',
-      creationDate,
+      creationDate.toString(),
```

Comment:
isn't it better to use toISOString() ?

Reply:
yes, it should work with all browsers, I agree. Solved

Reply:
I don't think so, it is an invalid date, so it is not properly initialized, and so, it crashes. 
Code to use as proof:
```
try {
      console.log(creationDate.toISOString())
    } catch (e) {
      console.log('prove that it crashes')
    }
```

and output:

```
console.log
    prove that it crashes

      at validateContribution (src/graphql/resolver/util/creations.ts:25:15)
```

Let me know if I am wrong, for the moment, I'm leaving it as it is